cmd/root.go

```diff
@@ -388,7 +392,11 @@ func updateIssue(ghIssue github.Issue, jIssue jira.Issue, ghClient github.Client
 				log.Infof("  Summary: %s", fields.Summary)
 			}
 			if fields.Description != "" {
-				log.Infof("  Description: %s...", fields.Description[0:20])
+				if len(fields.Description) > 20 {
```

Comment:
These changes seem unrelated to the dry-run option. Separate this into it's own PR so we can track and revert dry-run changes separately from this bug fix.